INSTALL.md

```diff
@@ -136,116 +177,109 @@ cmake -DCMAKE_PREFIX_PATH=<previous_build_dir>/dependencies/install
 ````
 
 ## [Building Antares Solution](#build)
-### Complete solution including GUI
+
 Antares source directory is named `[antares_src]` in all commands.
 
-Build can be done 'out of source'.
+Note :
+> cpack NSIS installer creation need a 'out of source build'. The build directory must be outside `[antares_src]` directory
+
+### Complete solution including GUI
 
-You can define build type with ```-DCMAKE_BUILD_TYPE``` option.
+You can define build type with ```-DCMAKE_BUILD_TYPE``` and ```--config``` option.
 
-release
+- release
 
 ```
-cmake -DCMAKE_BUILD_TYPE=release ..
+cmake -B _build -S [antares_src] -DCMAKE_BUILD_TYPE=release
+cmake --build _build --config release
 ```
- debug
+-  debug
  ```
-cmake -DCMAKE_BUILD_TYPE=debug ..
+cmake -B _build -S [antares_src] -DCMAKE_BUILD_TYPE=debug
+cmake --build _build --config debug
 ```
-Note that these are not the standard CMAKE_BUILD_TYPE. CMake files must be updated.
+Note :
+> These are not the standard ``CMAKE_BUILD_TYPE``. CMake files must be updated.
```

Comment:
If option ``CMAKE_BUILD_TYPE`` does not exist for your CMake, update it.

Reply:
This is not what I meant.
I'm saying that we are not using standard CMAKE_BUILD_TYPE. A feature must be developped to support standard values (Release Debug)

Reply:
ok